src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -66,19 +70,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
   }
 
   /**
-   * Build and return an authorized API client service.
-   *
-   * @return an authorized API client service
-   * @throws GeneralSecurityException, IOException
+   * Applies parameters to comment request, then us it to extract comments. URL is the only true
+   * variable; for this application we will always want order to be relevance, and max results to be
+   * 100, the API's arbitrary limit.
```

Comment:
We should remove the comment "The API's arbitrary limit" now that we know we could use a token to retrieve more comments.

Reply:
Agreed. I'll update it to reflect that it's the maximum that can be retrieved per token